Code/Engine/RendererCore/Lights/Implementation/LightComponent.cpp

```diff
@@ -60,18 +75,56 @@ ezColorGammaUB ezLightComponent::GetLightColor() const
   return m_LightColor;
 }
 
+ezColorGammaUB ezLightComponent::GetFinalLightColor() const
```

Comment:
If you do this inside GetLightColor, you should get the right color on the viewport icon.
Then you shouldn't need this function, at all.
If you need a function to get the 'raw' light color value, I would instead add a getter for that ("GetNonPhysicalLightColor" or something like that). But I don't really think you need that.

Reply:
fixed